packages/css/src/index.ts

```diff
@@ -368,7 +368,7 @@ export const css =
       const x = styles[key as keyof typeof styles]
       const val = typeof x === 'function' ? x(theme) : x
 
-      if (val && typeof val === 'object') {
```

Comment:
I agree that this is the best way to implement it, but I'm very conflicted about having a special case for `@page`.

It feels bad, doesn't it? Could we maybe... implement something else?
There's no perfect solution here, but I suppose I'd rather resign from the `size` shorthand and add `w` and `h` shorthands in its place.

An alternative we could consider — could we just not implement anything?
There's an escape hatch with `css` prop, Safari doesn't support `@page`. Seems less _correct_ than dropping the `size` alias, and we'd probably need an error thrown only in development, but it's something 🤷

Reply:
w/h shorthands will require changes by a significant percentage of Theme UI users & seems like a price they shouldn’t have to pay to me. The number of people using `@page` is likely tiny, & it’s very easy to update what code in the extremely rare case this change breaks it (imagine usages of that selector to find & replace vs “size” in a codebase). This isn’t very correct but it just doesn’t seem like a common issue to be sad about not getting your (screen-optimized) theme as automatically accessible in printing page setup for some browsers.